src/elements/content-preview/ContentPreview.js

```diff
@@ -686,6 +690,26 @@ class ContentPreview extends React.PureComponent<Props, State> {
         return !!showAnnotations && (this.canAnnotate() || hasViewAllPermissions || hasViewSelfPermissions);
     }
 
+    /**
+     * Returns whether a preview viewer supports printing
+     *
+     * @return {boolean}
+     */
+    handleCanPrint() {
+        /*
+        Setting this as default so older preview versions can still print documents without having the
+        default state have the print icon.
+         */
+        let canPrint = true;
```

Comment:
Would it make sense to default `canPrint` state to `true` then?

Reply:
Talked about this off-line, setting it to true will flash the icon and then remove it for files that cannot print.